src/netlink.c

```diff
@@ -549,6 +567,10 @@ static int qos_filter_cb (const struct nlmsghdr *nlh, void *args)
   mnl_attr_for_each (attr, nlh, sizeof (*tm))
   {
     struct gnet_stats_basic *bs = NULL;
+    struct gnet_stats_queue *qs = NULL;
+    struct qos_stats q_stats;
```

Comment:
This variable is called `qdisk` above and `q_stats` here. Please pick one and use the same name in both places.